packages/google_sign_in/lib/google_sign_in.dart

```diff
@@ -139,10 +135,12 @@ class GoogleSignIn {
   GoogleSignInAccount get currentUser => _currentUser;
 
   /// Attempts to sign in a previously authenticated user without interaction.
-  Future<GoogleSignInAccount> signInSilently() => _callMethod('signInSilently');
+  /// No-op if there is already a signed-in user.
```

Comment:
nit: first sentence of dart doc should be followed by an empty line.

Instead of calling it a no-op describe what happens if the user is already signed in (e.g. that user is returned).